setup.cfg

```diff
@@ -28,6 +28,9 @@ install_requires =
    pandas
    scipy
    scikit-learn
+   toolz
```

Comment:
Can these requirements be pushed down to e.g. an `experimental` option? Maybe the `wilds` section could be combined with that? With this config, a default install that doesn't plan to use any of the experimental code would pull in tensorflow and transformers, which IIRC are pretty big.